### ISSUE TITLE
possible to quiet output of ini() ?

### ISSUE THREAD

**Issue by oberon-manjaro** [Chrysostomus/aif-dev#112]
@papajoker as it is now, when ini() is invoked, dialog screen is pushed up and we can see for example ``[system]`` on the bottom of the screen.
Is it possible to divert/quiet the output?

**Comment by oberon-manjaro** [comment]
Ok. I have just checked.
Using 
```
ini_val "$INIFILE" "$section" "$value" &>/dev/null
```
works, without disturbing the resulting .ini file. Can we do that?

**Comment by papajoker** [comment]
yes and no, this &> we cant read a value `userfont=$(ini linux.font)` return nothing
I dont wiew outputs with ini()

a solution ?
```
if [ -n "$value"]
    ini_val "$INIFILE" "$section" "$value" &>/dev/null
else
    ini_val "$INIFILE" "$section" 2>/dev/null
fi
```

It would be cleaner to change in the original file ini_val.sh?

**Comment by oberon-manjaro** [comment]
> I dont wiew outputs with ini()

For me it looks like this currently:
![ini_system](https://cloud.githubusercontent.com/assets/11800714/23871818/41726a88-082c-11e7-8d9a-9c25105fd96b.png)

**Comment by papajoker** [comment]
fixed #9f444f1058a801720f699a0e8fc6f80b285341cc

**Comment by oberon-manjaro** [comment]
thanks! quiet now :)
